plugins/services/src/js/components/modals/NewCreateServiceModalServicePicker.js

```diff
@@ -1,144 +1,92 @@
-          headingClass={[
-            'panel-cell-borderless horizontal-center panel-cell-short',
-            {
-              'panel-cell-light': false,
-              'panel-cell-shorter': false
-            }
-          ]}
-          onClick={this.handleServiceSelect.bind(this, service)}>
-          <h5 className="flush text-align-center">
-            {title}
-          </h5>
-        </Panel>
-      </div>
-    );
-  }
+      <div className="create-service-modal-service-picker container">
+        <div className="create-service-modal-service-picker-options">
+          <div className="row">
+            <div className={containerClasses}>
+              <div className="row panel-grid">
+                <CreateServiceModalServicePickerOption
+                  onOptionSelect={
+                    this.handleServiceSelect.bind(null, {type: 'app'})
+                  }>
+                  <CreateServiceModalServicePickerOptionImage
+                    src={defaultServiceImage} />
+                  <CreateServiceModalServicePickerOptionContent>
+                    Single Container
+                  </CreateServiceModalServicePickerOptionContent>
+                </CreateServiceModalServicePickerOption>
 
-  getServiceOptionIconWrapper(icon) {
-    return (
-      <div className="icon icon-jumbo icon-image-container icon-app-container icon-default-white">
-        {icon}
-      </div>
-    );
-  }
+                <CreateServiceModalServicePickerOption
+                  onOptionSelect={
+                    this.handleServiceSelect.bind(null, {type: 'pod'})
+                  }>
+                  <CreateServiceModalServicePickerOptionImage
+                    src={defaultServiceImage} />
+                  <CreateServiceModalServicePickerOptionContent>
+                    Multi-container (Pod)
+                  </CreateServiceModalServicePickerOptionContent>
+                </CreateServiceModalServicePickerOption>
 
-  getUniversePackagesGrid() {
-    const packages = CosmosPackagesStore.getAvailablePackages();
-    let packagesGrid = packages.getItems().map((cosmosPackage, index) => {
-      return this.getServiceOption(
-        this.getServiceOptionIconWrapper(this.getPackageIcon(cosmosPackage)),
-        cosmosPackage.getName(),
-        index,
-        cosmosPackage
-      );
-    });
+                <CreateServiceModalServicePickerOption
+                  onOptionSelect={
+                    this.handleServiceSelect.bind(null, {type: 'json'})
+                  }>
+                  <CreateServiceModalServicePickerOptionImage
+                    src={jsonServiceImage} />
+                  <CreateServiceModalServicePickerOptionContent>
+                    JSON Configuration
+                  </CreateServiceModalServicePickerOptionContent>
+                </CreateServiceModalServicePickerOption>
 
-    return (
-      <div className="panel-grid row">
-        {packagesGrid}
+                <MountService.Mount
```

Comment:
How about rendering all service picker option with the mount component instead of hard coding a few?